app/views/includes/flash.njk

```diff
@@ -1,18 +1,18 @@
 {% from "../macro/success-notification.njk" import successNotification %}
 
 {% block flash %}
-  <div class="govuk-grid-column-two-thirds">
-    {% if flash.genericError %}
-    <div class="flash-container">
-      <div class="notification generic-error error-summary">
-        {{ flash.genericError | safe }}
+  {% if flash.genericError %}
+    <div class="govuk-grid-column-two-thirds">
+      <div class="flash-container">
+        <div class="notification generic-error error-summary">
+          {{ flash.genericError | safe }}
+        </div>
       </div>
+      {% endif %}
+      {% if flash.generic %}
+      {{ successNotification({
+        heading: flash.generic | safe
+      }) }}
     </div>
-    {% endif %}
-    {% if flash.generic %}
-    {{ successNotification({
-      heading: flash.generic | safe
-    }) }}
-    {% endif %}
-  </div>
+  {% endif %}
```

Comment:
Also, was the flash container causing issues with the display of the "My services" page? What happens if a notification is displayed to the user on the page?

Reply:
Good spot - I have fixed the `div` tags.

I noticed that the Flash container was always leaving an empty `<div class="govuk-grid-column-two-thirds">
  </div>` - even if there was not a Flash message.  I thought I would clean it up in case it cases any layout issues.

Now the `div` will only appear if there is a flash message.